cli/code_scanner.py

```diff
@@ -633,3 +723,8 @@ def _map_detections_per_file(detections) -> List[DetectionsPerFile]:
 
     return [DetectionsPerFile(file_name=file_name, detections=file_detections)
             for file_name, file_detections in detections_per_files.items()]
+
+
+def parse_commit_range(commit_range: str) -> (str, str):
+    commit_range_list = commit_range.split('...')
```

Comment:
commit range does not have to contain `...`

Reply:
Let's move it to a separate command. Cause in the SCA is not really scanning a range of commits. It scans diff between 2 commits